dist/forge-ui-components.js

```diff
@@ -5,6 +5,10 @@ var vue = require('vue');
 var script$1 = {
     name: 'ff-button',
     props: {
+        type: {
```

Comment:
@joepavitt I didn't check, and did run `git commit -a` 🙈 However, I think these files should be ignored. Will create a PR to do so.